src/PureScript/Backend/Chez/Syntax.purs

```diff
@@ -156,6 +156,9 @@ eqQ x y = runUncurriedFn (Identifier $ scmPrefixed "eq?") [ x, y ]
 vector :: Array ChezExpr -> ChezExpr
 vector = List <<< Array.cons (Identifier $ scmPrefixed "vector")
 
+hashtableCopy :: ChezExpr -> ChezExpr
+hashtableCopy h = runUncurriedFn (Identifier $ scmPrefixed "hashtable-copy") [ h ]
+
```

Comment:
Per [the `hashtable-copy` docs](https://scheme.com/tspl4/objects.html#./objects:s286), this form produces an immutable hashtable. So, does setting the copy's field even work?